src/data/markdown/docs/02 javascript api/07 k6-experimental/01 browser/close.md

```diff
@@ -1,11 +1,11 @@
 ---
 title: 'close()'
-excerpt: 'Browser module: Browser.close method'
+excerpt: 'Browser module: close method'
 ---
 
 Closes the browser and all of its pages (if any were opened).
 
-The [Browser](/javascript-api/k6-experimental/browser/browser-class/) object cannot be used anymore and is considered disposed of.
+Once closed, the [browser module API](/javascript-api/k6-experimental/browser#browser-module-api) cannot be used anymore, and the browser is considered disposed of.
```

Comment:
What will happen if one tries to use an already closed context? Shall we document this behavior?

Reply:
Yes, we display a warning message, https://github.com/grafana/xk6-browser/pull/803.

Reply:
Cool, let's maybe document this then?

Reply:
That makes sense. However, as I explained in [my other comment](https://github.com/grafana/k6-docs/pull/1200#discussion_r1223088215), the final API hasn't been concluded yet. So it's better to wait for it to evolve to a point where we can confidently update these parts of the doc.